diamm/serializers/search/organization.py

```diff
@@ -1,21 +1,21 @@
-from rest_framework import serializers
-from diamm.models.data.organization import Organization
+import serpy
 
 
-class OrganizationSearchSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = Organization
-        fields = ("type",
-                  "pk",
-                  "name_s",
-                  "location_s")
+class OrganizationSearchSerializer(serpy.Serializer):
 
-    # TODO: Find some way to refactor these into a base class for DRY
-    type = serializers.SerializerMethodField()
-    pk = serializers.ReadOnlyField()
+    type = serpy.MethodField()
+    pk = serpy.IntField()
 
-    name_s = serializers.ReadOnlyField(source="name")
-    location_s = serializers.ReadOnlyField(source="location.name")
+    location_s = serpy.StrField(
+        attr="location.name"
+    )
+    name_s = serpy.StrField(
+        attr="name",
+    )
+    organization_type_s = serpy.StrField(
+        attr="type",
+        required=False
+    )
```

Comment:
This is great, but it should be on a separate branch. Don't worry about it now, though -- just for future reference!

(You can always make an issue for things, e.g., "Rewrite Organization Search Serializer to use serpy" so you won't forget it...)